### DISCUSSION TITLE
Open URL Menu (session list)

### DISCUSSION THREAD

**Discussion by xavierve** [sienori/Tab-Session-Manager#1154] [Ideas]
In firefox, when I view the list of urls of a saved session, the context menu of each link (right clic on webpage Title) is capped. 
I think it would be very useful to have the full context menu as shown in any link, to be able to, for example:

Open link in private tab.
Open link in container ->
Share link with device. ->

Personally I'd love to, Open link in new temporary container.

Do you think it would be possible to add that functionality?

**Comment by sfederici** [comment]
> Share link with device. ->

Would this option give you a direct link to a session, so that it will open when put in Firefox's URL box?

**Comment by xavierve** [comment]
Each session has several urls
I am trying to work with one url of such session
1- go to sessions
2- click on one sessión (on sessions list, on the left)
3- shows all urls on that session, (urls list, on the right)
4- right click on one of those urls (an hyperlink with its title), and the contextual menu is not full, is not showing the normal commands when you right-click on a link
It laks of commands like:
- Save link to bookmark
- Send Link to Device / allows to share a link
- Open Link in New Window
- Open Link in Firefox Multi Container -> (work) (Personal) (buy)
- Open Link in Temporary Container / a plugin that auto deletes coockies, doesn't matter for what plugin is, the point is that LINK Contextual Menu is cut, not full, so we can do many things

thanks for your time in this :)
